lib/LaTeXML/Package/pgfplotstable.sty.ltxml

```diff
@@ -0,0 +1,22 @@
+# -*- mode: Perl -*-
+# /=====================================================================\ #
+# |  pgfplotstable.sty                                                  | #
+# | Implementation for LaTeXML                                          | #
+# |---------------------------------------------------------------------| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+#======================================================================
+InputDefinitions('pgfplotstable', type => 'sty', noltxml => 1)
+  || Warn(":missing:pgfplotstable.sty Couldn't find pgfplotstable.sty");
```

Comment:
isn't this wrong? 
InputDefinitions alsways returns without a value, so it would always be false and you'd always see the warning.
Also, if the file wasn't found, there is a `Warn()` inside InputDefinitions itself that will be emitted - I just tried that on my PR, you get the warning loud and clear.

Reply:
Ah I just spotted the `return $file;` in the case where InputDefinition succeeds. So it's correct to check the boolean, but redundant, since the `Warn('missing_file'` would get emitted in InputDefinition already.